src/CharCreation/Attributes.elm

```diff
@@ -69,7 +69,7 @@ viewAttribute attr model buttons =
     in
         div [ class "ui segments" ]
             [ div [ class "ui segment left aligned" ]
-                [ h4 [ class "ui header" ] [ text (toString attr) ]
+                [ h4 [ class "ui header" ] [ text ((toString attr) ++ ": " ++ (toString val)) ]
```

Comment:
maybe pull `toString attr` and `toString val` into their own definitions in the `let` block above?